java/client/src/main/java/glide/api/commands/servermodules/GlideJson.java

```diff
@@ -0,0 +1,411 @@
+        return executeCommand(
+                client,
+                new ArgsBuilder().add(gs(JSON_GET)).add(key).add(options.toArgs()).add(path).toArray());
+    }
+
+    /**
+     * Retrieves the JSON value at the specified <code>path</code> stored at <code>key</code>.
+     *
+     * @param client The Valkey GLIDE client to execute the command.
+     * @param key The <code>key</code> of the JSON document.
+     * @param paths List of paths within the JSON document.
+     * @param options Options for formatting the byte representation of the JSON data. See <code>
+     *     JsonGetOptions</code>.
+     * @return Returns a stringifies JSON object in bytes, in which each path is a key, and it's
+     *     corresponding value, is the value as if the path was executed in the command as a single
+     *     path. If <code>key</code> doesn't exist, returns null.
+     * @example
+     *     <pre>{@code
+     * String value = client.Json.set(client, "doc", new String[] {"$.a", "$.b"},
+     *                  JsonGetOptions.builder()
+     *                      .indent("  ")
+     *                      .space(" ")
+     *                      .newline("\n")
+     *                      .build()
+     *                  ).get();
+     * assert value.equals("{\n \"$.a\": [\n  1.0\n ],\n \"$.b\": [\n  2\n ]\n}");
+     * }</pre>
+     */
+    public static CompletableFuture<Object> get(
+            BaseClient client, String key, String[] paths, JsonGetOptions options) {
+        return executeCommand(
+                client,
+                ArrayTransformUtils.concatenateArrays(
+                        new String[] {JSON_GET, key}, options.toArgs(), paths));
+    }
+
+    /**
+     * Retrieves the JSON value at the specified <code>path</code> stored at <code>key</code>.
+     *
+     * @param client The Valkey GLIDE client to execute the command.
+     * @param key The <code>key</code> of the JSON document.
+     * @param paths List of paths within the JSON document.
+     * @param options Options for formatting the byte representation of the JSON data. See <code>
+     *     JsonGetOptions</code>.
+     * @return Returns a stringifies JSON object in bytes, in which each path is a key, and it's
+     *     corresponding value, is the value as if the path was executed in the command as a single
+     *     path. If <code>key</code> doesn't exist, returns null.
+     * @example
+     *     <pre>{@code
+     * GlideString value = client.Json.set(client, gs("doc"), new GlideString[] {gs("$.a"), gs("$.b")},
+     *                  JsonGetOptions.builder()
+     *                      .indent("  ")
+     *                      .space(" ")
+     *                      .newline("\n")
+     *                      .build()
+     *                  ).get();
+     * assert value.equals(gs("{\n \"$.a\": [\n  1.0\n ],\n \"$.b\": [\n  2\n ]\n}"));
+     * }</pre>
+     */
+    public static CompletableFuture<Object> get(
```

Comment:
Isn't **path** an optional parameter? Couldn't find a get() definition without path. Could you please tell if such a definition exists?

https://docs.aws.amazon.com/memorydb/latest/devguide/json-get.html

Reply:
Good catch. Will address this after the `String path` and `String[] paths` discussion is concluded.

Reply:
Just use `String[] paths` signature.  I don't think we are add much value by supporting the single String overload.